cfgov/paying_for_college/views.py

```diff
@@ -309,6 +311,33 @@ def school_search_api(request):
     return HttpResponse(json_doc, content_type='application/json')
 
 
+def school_autocomplete(request):
+    search_term = request.GET.get('q', '').strip()
+    if not search_term:
+        return HttpResponse(json.dumps([]), content_type='application/json')
```

Comment:
Could you remove some repeated use of `HttpResponse(json.dumps(...), content_type='application/json')` by using [`JsonResponse`](https://docs.djangoproject.com/en/3.1/ref/request-response/#jsonresponse-objects) instead?

```suggestion
        return JsonResponse([])
```

Reply:
I think we should switch to JsonResponse in most cases here, but when the data source is a list, I think we'll raise security hackles with the required safe=False.

Reply:
We could make sure all data payloads are dicts, but then some of the simplicity goes out the window, and we have to handle the extra unpacking in a template.

Reply:
Changed school_autocomplete to return a JsonResponse